labs/04-port-forward.md

```diff
@@ -36,12 +36,11 @@ sofus@Praq-Sof:~/git/docker-exercises$ docker container run -p 8080:80 nginx
 
 Press **control + c** in your terminal window to stop your container.
 
-## Working with your docker container
+## Running the webserver container in the background
 
-When running a webserver like nginx, it's pretty useful that you do not have to have an open session into the server at all times to run it.
-We need to make it run in the background, freeing up our terminal for other things.
-Docker enables this with the `-d` parameter for run.
-`docker container run -p 8080:80 -d nginx`
+When running a webserver like nginx, it is very useful to not run the container in the foreground of your terminal, instead we should make it run in the background, freeing up our terminal for other things.
```

Comment:
Very long sentence. Would it make sense to break it up in the middle?
`[...]foreground of your terminal. Instead we should[...]`

Reply:
Done in 2b4d547